Gemfile

```diff
@@ -2,10 +2,10 @@ source 'https://rubygems.org'
 
 ruby File.read(".ruby-version")
 
-gem 'rails', '~> 5.1.5'
+gem 'rails', '~> 5.1.6'
 gem 'pg'
 gem 'puma'
-gem 'devise', '~> 4.4.0'
+gem 'devise'
```

Comment:
Well, I don't really know if I'm actually following a _"standard"_ here 😉 

Before I go to update gems, I look at the changelog for every gem where a new version has been released to check if there are breaking changes. If there aren't any, I just update all gems at once with `bundle update` and see if things break afterwards.
However, if there are certain breaking changes that require enough changes on our side that I would not shove it into a PR simply called "update dependencies", I lock it to the newest version compatible with our application code, so that we don't accidentally update it to a version that would break our app.

For this particular case, I could not find a reason why we wanna lock `devise` to this particular version - so I removed the lock.

Reply:
Sounds good to me, thanks for clarifying, @klappradla!